test_data/invalid_feed_name_in_exit_condition.yaml

```diff
@@ -0,0 +1,11 @@
+state_machines:
+  the_workflow:
+    feeds:
+      - name: jacquard
+        url: http://localhost:1212/<label_id>
+    states:
+      - gate: choice
+        triggers:
+          - event: entry
+        # This should be `feeds.jacquard.has_option_b`
```

Comment:
Not sure if this reads too much like a TODO rather than documenting the point of the test, what do you think? Happy to leave as-is, but if you think of a better wording do change.

Reply:
Yup, clarified in 4ab691a.